config/routes.rb

```diff
@@ -81,6 +81,7 @@
   get '/export/full_history_patient/:patient_id', to: 'export#full_history_patient'
   post '/export/custom', to: 'export#custom_export'
   get '/export/nbs/patient/:patient_id', to: 'export#nbs_patient'
+  get '/export/nbs/patient', to: 'api_export#nbs_patients'
```

Comment:
This is the first non-FHIR API route, so yeah I think it does make sense to nest it as you said (and to nest the controller within a folder).

Reply:
+1

Reply:
You think if it is in a namespace we still need `export/nbs/patient`, so it would be `api/export/nbs/patient`? I am thinking `export` isn't really necessary anymore.

Reply:
Done in https://github.com/SaraAlert/SaraAlert/pull/823/commits/ead02e3fc1bcc79b2c717085f526b9b2cf59c480. I went with `api/nbs/patient` for now. Let me know if that needs to change.

Reply:
Totally agree with api/nbs/patient .. it's  GET request so with that name I expect it to return in that format.